src/c_api/c_api_utils.h

```diff
@@ -12,10 +12,11 @@
 #include <vector>
 
 #include "xgboost/c_api.h"
-#include "xgboost/data.h"  // DMatrix
+#include "xgboost/data.h"         // DMatrix
+#include "xgboost/feature_map.h"  // for FeatureMap
```

Comment:
The `feature map` is no longer included in `learner.h`. This is to fix missing includes.